File: tests/fixtures/vulnerable/js/command-injection.js
// Test fixture: Command injection
// Should trigger: command-injection

const { exec, execSync } = require('child_process');

function pingHost(host) {
  exec(`ping -c 1 ${host}`, (err, stdout) => {
    console.log(stdout);
  });
}

function processFile(filename) {
  const result = execSync(`cat ${filename}`);
  return result.toString();
}

module.exports = { pingHost, processFile };

File: tests/fixtures/vulnerable/py/command-injection.py
# Test fixture: Command injection
# Should trigger: command-injection

import os
import subprocess

def ping_host(host):
    os.system(f"ping -c 1 {host}")

def process_file(filename):
    subprocess.run(f"cat {filename}", shell=True)

def backup_data(path):
    os.system(f"tar -czf backup.tar.gz {path}")
